python/ray/data/_internal/planner/plan_read_op.py

```diff
@@ -36,6 +38,38 @@ def cleaned_metadata(read_task):
     return block_meta
 
 
+def _splitrange(n, k):
+    """Calculates array lens of np.array_split().
+
+    This is the equivalent of
+    `[len(x) for x in np.array_split(range(n), k)]`.
+    """
+    base = n // k
+    output = [base] * k
+    rem = n - sum(output)
+    for i in range(len(output)):
+        if rem > 0:
+            output[i] += 1
+            rem -= 1
+    assert rem == 0, (rem, output, n, k)
+    assert sum(output) == n, (output, n, k)
+    return output
+
+
+def _do_additional_splits(
+    blocks: Iterable[Block], _: TaskContext, additional_output_splits: int
+) -> Iterable[Block]:
+    """Do additional splits to the output blocks of a ReadTask."""
```

Comment:
nit: does it matter that `blocks` are the output of a `ReadTask`?

Reply:
The implementation doesn't matter. But the purpose of this function is to process the output blocks of a read task. So I prefer to say that in the comment.